code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -2,13 +2,42 @@
     openapi: 3.0.3
     info:
       title: Connectivity Insights API
-      description: Find out the likelihood of the network meeting the networking requirements of your application for a given user session,.
       version: '0.3.0-wip'
       contact: 
        email: sp-coi@lists.camaraproject.org
       license:
         name: Apache 2.0
         url: https://www.apache.org/licenses/LICENSE-2.0.html
+      description: |
+        Connectivity Insights API provides programmable interface for developers and other users (capabilities consumers) with the much needed visibility into the network performance data in the form of an easy to understand and use intent based API.
```

Comment:
I think the first two paragraphs are too 'marketing-speak'. How about the paragraph I suggested followed by key points from  your 2nd and 3rd paragraph, i.e.:
"
The Connectivity Insights API allows an application developer to ask the network the likelihood that an  application's networking requirements can be met for a given end user session.

This information helps a developer  ensure their end users are able to get the best experience while using the application over their current network. 
        Depending on the answer the network gives, the developer may decide to request a network boost (via the CAMARA QoD API) , and/or apply specific changes on the application side e.g. adjusting the resolution of the video stream upwards or downwards."

Reply:
incorporated this feedback and by incorporating the suggested text.